src/metric-engine/src/engine/drop.rs

```diff
@@ -71,7 +71,7 @@ impl MetricEngineInner {
                 .get(&region_id)
                 .copied();
             if let Some(metadata_region_id) = metadata_region_id {
-                self.drop_logical_region(region_id, metadata_region_id)
+                self.drop_logical_region(region_id, metadata_region_id, req.fast_drop_path)
```

Comment:
The `force_drop_all_logical_tables` flag should only apply to the physical region